app/src/main/java/de/berlindroid/zepatch/ui/PatchableToBitmap.kt

```diff
@@ -20,15 +22,25 @@ import de.berlindroid.zepatch.utils.CaptureToBitmap
 @Composable
 fun PatchableToBitmap(
     modifier: Modifier = Modifier,
+    onBitmap: (ImageBitmap) -> Unit = {},
     patchable: @Composable () -> Unit,
 ) {
     var image by remember { mutableStateOf<ImageBitmap?>(null) }
+    var capture by remember { mutableStateOf(false) }
```

Comment:
Naming: Do we mean `captured` (mind the `d`) here? I.e. was the image captured, or is that meant as a "It should be captured soon?"

Reply:
I renamed it to shouldCapture